tests/integration/test_pulls.py

```diff
@@ -139,6 +147,31 @@ def test_repository(self):
             assert p.repository == ('sigmavirus24', 'github3.py')
 
 
+class TestPullReview(IntegrationHelper):
+    """Integration tests for the PullFile object."""
+    def get_pull_request_review(self, owner, repo, pull_number, review):
+        p = self.gh.pull_request(owner, repo, pull_number)
+
+        for pull_review in p.reviews():
+            if pull_review.id == review:
+                break
+        else:
+            assert False, "Could not find '{0}'".format(review)
+
+        return pull_review
+
+    def test_user(self):
+        """Show that a user can retrieve the user of a PR review."""
+        cassette_name = self.cassette_name('pull_reviews')
+        with self.recorder.use_cassette(cassette_name):
```

Comment:
I don't know how to properly record one, so if you could, that'd be great! (or point out the docs for how to get it recorded, that would be okay as well)